### ISSUE TITLE
Cannot Create Team after New Deployment to AWS 

### ISSUE THREAD

**Issue by gdziuba** [FlowFuse/flowfuse#3966]
### Current Behavior

When deployed FlowFuse to AWS via the AWS Marketplace, I wasn't able to create a team after it started.  I was able to fix this by restarting the the EC2 instance.  Once restarted, the docker daemon wasn't running.  Had to start that again and everything functioned as expected.

Here is an error message that I received after initializing the server and initiating the instance when loading my DNS info.

```<!doctype html><html lang=""><head><link rel="icon" href="/favicon.ico"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>FlowFuse</title><script>/*inject-ff-scripts*/</script><link rel="icon" href="/app/favicon.ico"><script defer="defer" src="/app/runtime.js"></script><script defer="defer" src="/app/vendors.js"></script><script defer="defer" src="/app/main.js"></script><link href="/app/main.css" rel="stylesheet"></head><body class="bg-white"><noscript><strong>We're sorry but FlowFuse doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id="app"></div></body></html>```

### Expected Behavior

When I created a team, I expected it to transition to the next step and allow me to create an instance.

### Steps To Reproduce

Deploy from here: https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa#pdp-overview

Go through the [traditional steps](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa#pdp-overview)

I don't know if it is relevant, but I did have issues when creating a new password.  I doubt that is relevant but thought I would share anyways.

Next, I was brought with a screen to create the team.  Once creating a new team name and submitting, it didn't transition to the next screen.

### Environment

- FlowFuse version:
- Node.js version:
- npm version:
- Platform/OS:
- Browser:

### Have you provided an initial effort estimate for this issue?

I can not provide an initial effort estimate

**Comment by ZJvandeWeg** [comment]
@hardillb Can you check this out?

**Comment by hardillb** [comment]
It really isn't clear what the error message you mean here. The following output is totally expected (but I admit we probably could hide it)

```
...
Waiting 30 seconds for startup

********************************************************************************
<!doctype html><html lang=""><head><link rel="icon" href="/favicon.ico"><link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png"><link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>FlowFuse</title><script>/*inject-ff-scripts*/</script><link rel="icon" href="/app/favicon.ico"><script defer="defer" src="/app/runtime.js"></script><script defer="defer" src="/app/vendors.js"></script><script defer="defer" src="/app/main.js"></script><link href="/app/main.css" rel="stylesheet"></head><body class="bg-white"><noscript><strong>We're sorry but FlowFuse doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id="app"></div></body></html>
********************************************************************************

You can then finish setting up your FlowForge instance at

https://forge.aws.flowforge-demo.co.uk/setup

Username: ff-admin
...
```

I've just tested the latest image and it runs just fine:

 - I signed in with the presented username/password
 - Changed the password
 - created a team
 - created an application/instance

I didn't have to restart anything.

But the docker daemon is set to automatically start on boot, so it will start, but I have no idea how long you waited after restarting the instance.

**Comment by hardillb** [comment]
Yep, just restarted the EC2 instance and when I logged back in Docker was running, as were all the containers
